plugins/get-all.yml

```diff
@@ -10,4 +10,4 @@ plugin:
     background: false
     args:
     - -c
-    - "kubectl get-all -n $NAMESPACE | less"
\ No newline at end of file
+    - "kubectl --context $CONTEXT get-all -n $NAMESPACE | less"
```

Comment:
I think it should be `kubectl get-all --context ...`

Reply:
@derailed both syntax works, I can make the required change if you prefer ?